staging/src/k8s.io/apimachinery/pkg/runtime/scheme.go

```diff
@@ -575,6 +575,24 @@ func (s *Scheme) generateConvertMeta(in interface{}) (conversion.FieldMatchingFl
 	return s.converter.DefaultMeta(reflect.TypeOf(in))
 }
 
+// Creates and returns a copy of Scheme.typeToGVK
+func (s *Scheme) ExportTypesToGVKMaps() map[reflect.Type][]schema.GroupVersionKind {
+	ret := map[reflect.Type][]schema.GroupVersionKind{}
+	for k, v := range s.typeToGVK {
+		ret[k] = v
+	}
+	return ret
+}
+
+// Creates and returns a copy of Scheme.unversionedTypes
+func (s *Scheme) ExportUnversionedTypesMap() map[reflect.Type]schema.GroupVersionKind {
```

Comment:
I don't think we need to add these methods since you can already get them via traversal?